testing/dart/observatory/vmservice_methods_test.dart

```diff
@@ -41,4 +40,68 @@ void main() {
       await vmService?.dispose();
     }
   });
+
+  test('Reload fonts request sends font change notification', () async {
+    vms.VmService? vmService;
+    try {
+      final developer.ServiceProtocolInfo info =
+          await developer.Service.getInfo();
+      if (info.serverUri == null) {
+        fail('This test must not be run with --disable-observatory.');
+      }
+
+      final Completer<PlatformResponse> completer = Completer<PlatformResponse>();
+      ui.window.onPlatformMessage = (String name, ByteData? data, ui.PlatformMessageResponseCallback? callback) {
+        final ByteBuffer buffer = data!.buffer;
+        final Uint8List list = buffer.asUint8List(data.offsetInBytes, data.lengthInBytes);
+        completer.complete(PlatformResponse(name: name, contents: utf8.decode(list)));
+      };
+
+      vmService = await vmServiceConnectUri(
+        'ws://localhost:${info.serverUri!.port}${info.serverUri!.path}ws',
+      );
+      final String viewId = await getViewId(vmService);
+
+      final vms.Response fontChangeResponse = await vmService.callMethod(
+        '_flutter.reloadAssetFonts',
+        args: <String, Object>{'viewId': viewId},
+      );
+
+      expect(fontChangeResponse.type, 'Success');
+      expect(
+        await completer.future.timeout(const Duration(minutes: 3)),
```

Comment:
🤷‍♂️

Reply:
Why not forever?

Reply:
Don't know, worried about what happens if this test breaks. Does the entire shard time out at 60 minutes?